App.js

```diff
@@ -1,12 +1,13 @@
-import Dodge from "./Dodge.js";
+import DodgeBallGame from "./Dodge.js";
```

Comment:
Maybe this eventually lives in `games/DodgeBall.js`.

[Folders are typically `lowercase_with_underscores`, and files within folders don't need to repeat information that's clear from the folder path (i.e. `mini_games/DodgeBallGame.js` would be redundant)].

Reply:
when attempting to do that now I get the error message:
Uncaught SyntaxError: Unexpected token '!' (at bundle.js:1408:25)
which to me seems to be saying that the folder doesn't have access to one thing or another